layer1/layer1.yaml

```diff
@@ -10,124 +10,112 @@ components:
       description: >-
         A container for layer1 settings.
       type: object
-      required: [name, choice]
+      required: [name, port_names]
       properties:
         name: '../common/common.yaml#/components/x-inline/UniqueName'
         port_names:
           description: >- 
-            A list of unique names of a port objects that will share the 
+            A list of unique names of port objects that will share the
             choice settings. 
           type: array
           items:
             type: string
             x-constraint:
             - '/components/schemas/Port/properties/name'
-        choice:
+        speed:
           description: >-
-            The type of layer1 characteristics.
+            Set the speed if supported.
           type: string
-          enum: [one_hundred_gbe, ethernet, ethernet_virtual]
-        one_hundred_gbe:
-          $ref: '#/components/schemas/Layer1.OneHundredGbe'
-        ethernet:
-          $ref: '#/components/schemas/Layer1.Ethernet'
-        ethernet_virtual:
-          $ref: '#/components/schemas/Layer1.EthernetVirtual'
-
-    Layer1.OneHundredGbe:
-      description: >-
-        Container for 100 gigabit ethernet settings
-      type: object
-      properties:
-        ieee_media_defaults:
+          enum:
+            - speed_10_mbps
+            - speed_100_mbps
+            - speed_1_gbps
+            - speed_10_gbps
+            - speed_25_gbps
+            - speed_40_gbps
+            - speed_100_gbps
+            - speed_200_gbps
+            - speed_400_gbps
+          default: speed_10_gbps
+        media:
           description: >-
-            Enable/disable ieee media default settings.
-            True will override the speed, auto_negotiate, link_training, rs_fec settings.
-          type: boolean
-          default: true
-        auto_negotiate:
+            Set the type of media interface if supported.
+          type: string
+          enum:
+            - copper
+            - fiber
+            - sgmii
+        promiscuous:
           description: >-
-            Enable/disable auto negotiation.
+            Enable promiscuous mode if supported.
           type: boolean
           default: false
-        link_training:
+        mtu:
           description: >-
-            Enable/disable link training.
-          type: boolean
-          default: false
-        rs_fec:
+            Set the maximum transmission unit size if supported.
+          type: integer
+          default: 1500
+        ieee_media_defaults:
           description: >-
-            Enable/disable reed solomon forward error correction (RS FEC).
+            Set to true to override the auto_negotiate, link_training
+            and rs_fec settings for gigabit ethernet interfaces.
           type: boolean
-          default: false
-        speed:
+          default: true
+        auto_negotiate:
```

Comment:
We don't need this anymore (since all AN-related settings should now be part of AutoNegotiation)